addons/web/static/tests/views/kanban/kanban_view.test.js

```diff
@@ -10220,35 +9896,23 @@ test("test displaying image (binary & placeholder)", async () => {
         resModel: "partner",
         arch: `
             <kanban>
-                <field name="id"/>
-                <field name="image"/>
                 <templates>
-                    <t t-name="kanban-box">
-                        <div>
-                            <img t-att-src="kanban_image('partner', 'image', record.id.raw_value)"/>
-                        </div>
+                    <t t-name="kanban-card">
+                        <field name="image" widget="image" />
                     </t>
                 </templates>
             </kanban>`,
     });
 
     expect(queryAll(`.o_kanban_record img`).map((img) => img.dataset.src.split("?")[0])).toEqual([
         "data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACAA==",
-        `${getOrigin()}/web/image/partner/2/image`,
-        `${getOrigin()}/web/image/partner/3/image`,
-        `${getOrigin()}/web/image/partner/4/image`,
+        `/web/static/img/placeholder.png`,
+        `/web/static/img/placeholder.png`,
+        `/web/static/img/placeholder.png`,
     ]);
 });
 
 test("test displaying image (for another record)", async () => {
```

Comment:
this one is no longer relevant (not supported anymore)

Reply:
actually I think we can remove all tests "test displaying image..." as the feature is no longer a kanban feature. We simply use the image field, which is already tested in its own file.